RobotFramework_TestsuitesManagement/Config/CConfig.py

```diff
@@ -587,13 +588,13 @@ def __loadConfigFileLevel2(self) -> bool:
         if sTestCfgDir.startswith('.../'):
             sTestCfgDirStart = sTestCfgDir
             sTestCfgDir = sTestCfgDir[4:]
```

Comment:
Current code:
```
sTestCfgDir = sTestCfgDir[4:]
if os.path.exists(CString.NormalizePath('./' + sTestCfgDir)):
    sTestCfgDir = './' + sTestCfgDir
```

Why not
```
sTestCfgDir = sTestCfgDir[2:] # '2:' skips the first two dots in '.../'
if os.path.exists(CString.NormalizePath(sTestCfgDir)):
```
?

When you use CString.NormalizePath to normalize a relative path, you urgently should use 'sReferencePathAbs' to
explicitely define the (absolute) path that shall be the reference for the relative path.
Otherwise the operating system will decide. Then you loose the control!

An example you can find in
robotframework-testsuitesmanagement\config\CRepositoryConfig.py
Search for: `sReferencePathAbs=self.__sReferencePath`

Reply:
Hello Holger,

We use sTestCfgDir = sTestCfgDir[4:] to remove '.../' because in else case of condition 'if os.path.exists(CString.NormalizePath('./' + sTestCfgDir)):' we have the code "sTestCfgDir = '../' + sTestCfgDir" to check an upper directory.

You can look into the code block form line 588 to 605.

Thank you,
Son